doc/reference/std/text/json.md

```diff
@@ -121,6 +121,46 @@ If bound to `list->vector` then JSON lists will be parsed as vectors.
 #("a" 1)
 ```
 
+## json-sort-keys
+``` scheme
+json-object-keys
+```
+
+This is a parameter that can be used to control how JSON objects should be encoded.
+If true (the default), keys in JSON objects represented by hash-tables will be sorted
```

Comment:
false is a more reasonable default i think.

Reply:
True is the current default, I suppose for the sake of determinism. It's still time to change it before the next release.

Reply:
Yeah, I think we should.